### PR TITLE
Improve an assert message in DoFAccessorImplementation

### PR DESCRIPTION
**Pull request by peterrum**
references #12534

FYI @bangerth

### CODE REVIEW

**Review by kronbichler — APPROVED** [review]

File: include/deal.II/dofs/dof_accessor.templates.h [hunk 1]
```diff
@@ -286,7 +286,10 @@ namespace internal
 
         Assert(ptr != dof_handler.hp_object_fe_indices[d].begin() +
                         dof_handler.hp_object_fe_ptr[d][obj_index + 1],
-               ExcNotImplemented());
+               ExcMessage(
+                 "You are requesting from this entity an "
+                 "active_fe_index that is not assigned to any of the cells "
+                 "connected to this entity."));
```
Comment (drwells) [inline include/deal.II/dofs/dof_accessor.templates.h:292]:
```suggestion
                 "You are requesting an active_fe_index that is not assigned "
                 "to any of the cells connected to this entity."));
```

**Review by drwells — CHANGES_REQUESTED** [review]
We can make the sentence clearer.

File: include/deal.II/dofs/dof_accessor.templates.h (hunk 1 above)
Comment (peterrum) [reply include/deal.II/dofs/dof_accessor.templates.h]: I have changed the text. Good to have a native speaker in the team :+1:

**Review by drwells — APPROVED** [review]

**Review by bangerth — APPROVED** [review]

**Pull request merged**